server/controllers/finance/reports/cashflow.report.handlebars

```diff
@@ -0,0 +1,212 @@
+<head>
+  <meta charset="UTF-8">
+  <title>{{translate "TREE.CASHFLOW"}}</title>
+  <link href="{{absolutePath}}/css/bhima-bootstrap.css" rel="stylesheet">
```

Comment:
This is a partial that does this (and more) called `head`.   It's syntax is like this:

```hbs
{{> head title="TREE.CASHFLOW" }}
```
Is there a reason why you cannot use it?